src/frontend/next/src/components/SEO.tsx

```diff
@@ -0,0 +1,27 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import config from '../config';
+
+type SEOProps = {
+  pageTitle: string;
+};
+
+const SEO = ({ pageTitle }: SEOProps) => {
+  const { telescopeUrl } = config;
+  const { pathname } = useRouter();
+  const currentUrl = `${telescopeUrl}${pathname}`;
+
+  return (
+    <Head>
+      <meta
+        name="viewport"
+        content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no, viewport-fit=cover"
+      />
+      <meta property="og:url" content={currentUrl} key="ogurl" />
```

Comment:
Why do you include `key`? in these?

Reply:
@humphd : I read from the link that Pedro provided that key would make the meta unique, but I guess it is not necessary here.

Reply:
It's necessary if you're building it via a loop with React, yes, since React needs to know how to associate the DOM node with the data in the array; but not in this case.